src/pages/storage/forms/StoragePoolClusteredSourceSelector.tsx

```diff
@@ -0,0 +1,40 @@
+import { Label } from "@canonical/react-components";
+import ClusterSpecificInput from "components/forms/ClusterSpecificInput";
+import { FormikProps } from "formik";
+import { FC } from "react";
+import { StoragePoolFormValues } from "./StoragePoolForm";
+import { useClusterMembers } from "context/useClusterMembers";
+
+interface Props {
+  formik: FormikProps<StoragePoolFormValues>;
+  helpText: string;
```

Comment:
```suggestion
  helpText?: string;
```
I think this should be optional.